### ISSUE TITLE
"check" dependency missing?

### ISSUE THREAD

**Issue by dasdeck** [gadicc/meteor-messageformat#188]
Hey there!
Out of the box I got the following issue:
```
I20160105-13:51:30.741(1)? Exception from sub mfStrings id EyFEHDjrYCERMGgZ8 ReferenceError: Match is not defined
I20160105-13:51:30.741(1)?     at [object Object].Meteor.publish.query [as _handler] (packages/gadicohen_messageformat/packages/gadicohen_messageformat.js:2499:1)
I20160105-13:51:30.742(1)?     at maybeAuditArgumentChecks (livedata_server.js:1698:12)
I20160105-13:51:30.742(1)?     at [object Object]._.extend._runHandler (livedata_server.js:1023:17)
I20160105-13:51:30.742(1)?     at [object Object]._.extend._startSubscription (livedata_server.js:842:9)
I20160105-13:51:30.743(1)?     at [object Object]._.extend.protocol_handlers.sub (livedata_server.js:614:12)
I20160105-13:51:30.744(1)?     at processNext (livedata_server.js:548:43)

```

running...
```
meteor add check
```

...fixed the issue.

You might want to add it as a dependecy?
I could certainly do it myself though.

Cheers!

JM

**Comment by DSpeichert** [comment]
Is that for v0 or v2?

**Comment by dasdeck** [comment]
hmmm latest meteor & meteor add gadicohen:messageformat ?
I'll have to find out....

**Comment by DSpeichert** [comment]
That would be v0.

**Comment by gadicc** [comment]
Hi guys, thanks for the bug report and version clarification.

From what I recall this affects new (not upgraded) Meteor 1.2 installations, from their stricter global imports.  Can you confirm?

Either way, I think despite v0 being frozen, it's nice to get this fixed, which I've done in `v0.0.53`.
